pkg/kustomize/kustomize.go

```diff
@@ -0,0 +1,202 @@
+/*
```

Comment:
Please add a test for this file. In the test, you can use the fact the test working directory is set to the `_test.go` directory. And then you use `http.Dir` for accessing the test files (it implements the `http.FileSystem` interface, read the docs). No need for statik for test cases.